docs/data/material/pages.ts

```diff
@@ -19,6 +20,12 @@ const pages: MuiPage[] = [
       { pathname: '/material-ui/getting-started/support' },
     ],
   },
+  {
+    title: 'Varnish',
+    pathname: '/material-ui/varnish',
```

Comment:
```suggestion
    pathname: '/allenai/varnish',
```

I know, I'm a stickler about names. But this should help avoid collissions with MUI stuff.

Reply:
hmm...
the code need to exist in /material-ui
and paths are odd in next... im thinking that from a user perspective it will be
varnish.allenai.org/material-ui/varnish <- varnish specific components

heck, maybe it should be renamed to `/material-ui/allenai` 
varnish.allenai.org/material-ui/allenai

or `/material-ui/ai2-components`
varnish.allenai.org/material-ui/ai2-components

Reply:
This might be reason to do the path munging in a proxy that sits in front of the site. That way the end user never sees these paths. We'd just treat `/material-ui` as the root.

So the'd see paths like`/varnish` and `/getting-started/installation`, which would get prefixed with `/material-ui` by the proxy.